src/buyflow/SummaryStep.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+interface SummaryStepProps {
+  collectedData: {
+    email: string
+    age: number
+  }
+}
+
+const SummaryStep: React.FC<SummaryStepProps> = (props) => {
+  return (
+    <>
+      <div>Email: {props.collectedData.email}</div>
+      <div>Age: {props.collectedData.age}</div>
```

Comment:
most of the `...Step` components are focused on only one thing, but this one assumes the shape of the collected data, let's make sure that it can work with different cases:
- props type should become more generic
- body should map over `collectedData` to show it in a dynamic manner